components/bluetooth_proxy.rst

```diff
@@ -22,6 +22,9 @@ our `Bluetooth Proxy installer <https://esphome.github.io/bluetooth-proxies/>`__
     Passively broadcasted sensor data (that is advertised by certain devices without active connections) is received
     separately from these, and is not limited to a specific number.
     
+    When using active connections, the :doc:`web_server` component should be disabled as the device is likely
+    to run of memory and malfunction when both components are enabled simultaneously.
+    
```

Comment:
I would even go as far as to make this whole thing its own warning box

```suggestion

  .. warning::

    
    When using active connections, the :doc:`web_server` component should not be configured as the device is likely
    to run out of memory and malfunction when both components are enabled simultaneously.
    This can also be the case when trying to add ``bluetooth_proxy`` to a device that already has a number of other 
    components configured. It is best to make the device as dedicated to ``bluetooth_proxy`` as you can to 
    have the best results.
    
```

Reply:
Adjusted